docs/cli/usage/iam.md

```diff
@@ -118,6 +118,7 @@ The Amplify CLI requires the below IAM policies for performing actions across al
                 "cognito-idp:ListUserPoolClients",
                 "cognito-idp:UpdateUserPool",
                 "cognito-idp:UpdateUserPoolClient",
+                "cognito-idp:UpdateGroup",
```

Comment:
It looks like this is out of sort order.

Reply:
Fixed